clang/test/dpct/cmake_migration/case_013/input.cmake

```diff
@@ -0,0 +1,16 @@
+project(foo-bar LANGUAGES CXX CUDA)
+
+set(cufile file1.cu)
+cuda_compile_cubin(cubinfile_1 ${cufile})
+cuda_compile_cubin(
+  cubinfile_2 
+  ${cufile}
+)
+
+cuda_compile_cubin(cubinfiles_1 file1.cu file2.cu)
+cuda_compile_cubin(cubinfiles_2 
+  file1.cu 
+  file2.cu)
+
+cuda_compile_cubin(cubinfile_1 file1.cu OPTIONS
+  -O3 -Xptxas -v --use_fast_math)
```

Comment:
What happens to the `nvcc` options not supported by the `icpx`?

Reply:
Options will be removed